Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -55,6 +55,12 @@ open class NavigationMapView: UIView {
     public var roadClassesWithOverriddenCongestionLevels: Set<MapboxStreetsRoadClass>? = nil
     
     var cameraAnimator: CameraAnimator!
+    /**
+     Controls whether to show congestion levels on alternative route lines. Defaults to `false`.
+     
+     If `true` and there're multiple routes to choose, the alternative route lines would display the congestion levels at different colors, similar to the main route. To customize the congestion colors that represent different congestion levels, override the `alternativeTrafficUnknownColor` , `alternativeTrafficLowColor`, `alternativeTrafficModerateColor`, `alternativeTrafficHeavyColor`, `alternativeTrafficSevereColor` property for the `NavigationMapView.appearance()`.
+     */
+    public var showsAlternativeCongestion: Bool = false
```

Comment:
IMO `showsCongestionForAlternativeRoutes` would be more descriptive.

Reply:
Thanks for your suggestion! I'll update this part.